aptos-move/move-examples/my_first_dapp/move/sources/main.move

```diff
@@ -0,0 +1,121 @@
+module myaddr::main {
+
+  use aptos_framework::account;
+  use std::signer;
+  use aptos_framework::event;
+  use std::string::String;
+  use aptos_std::table::{Self, Table};
+  #[test_only]
+  use std::string;
+
+  // Errors
```

Comment:
Can you add comments for these errors? These comments are used to generate human readable error messages for tx failures. It might be worth mentioning this in the tutorial as well

Reply:
let me add it in a second PR after that one lands. it involves changing steps in the doc also.

Reply:
I think we should still add this here if this is meant to be used as a template. This can remind devs that these error comments are required for generating human-readable errors